Scripts/RemediationScripts/ControlsEligibleForRemediationThroughUI.md

```diff
@@ -29,6 +29,8 @@
 
 14. [Azure_SQLDatabase_DP_Use_Secure_TLS_Version_Trial](ControlsEligibleForRemediationThroughUI.md#14-Azure_SQLDatabase_DP_Use_Secure_TLS_Version_Trial)
 
+15. [Azure_Storage_DP_Use_Secure_TLS_Version_Trial](ControlsEligibleForRemediationThroughUI.md#14-Azure_Storage_DP_Use_Secure_TLS_Version_Trial)
```

Comment:
Change 14 --> 15 in "...ThroughUI.md#14-Azure_Sto..."
It's making the click feature on markdown not work.

Reply:
Changed